app/views/layouts/_header.html.erb

```diff
@@ -9,8 +9,10 @@
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav mr-auto">
         <li class="nav-item active background-blue">
-          <%= link_to new_user_registration_path, class: 'nav-link nav-icon nav-first' do %>
-            <%= image_tag "BW- CREATE USER ICON.svg", alt:"Create a User" %><p>CREATE USER</p>
+          <% if current_user&.role == 'admin' || current_user&.role == 'volunteer' %>
```

Comment:
You can use `current_user.admin? || current_user.volunteer?`  Rails enum gives us these nice helper methods :)

Reply:
or we could reuse this helper https://github.com/rubyforgood/babywearing/blob/master/app/helpers/users_helper.rb#L8

Reply:
Very nice! Thank you!

Reply:
Well I had to add the safe navigation in your helper method, can it be a future problem?